frontend/packages/operator-lifecycle-manager/src/components/modals/uninstall-operator-modal.tsx

```diff
@@ -65,9 +96,36 @@ export const UninstallOperatorModal: React.FC<UninstallOperatorModalProps> = ({
         })
         .catch(() => {});
     },
-    [close, handlePromise, k8sKill, subscription],
+    [close, deleteAllOperands, handlePromise, k8sKill, models, operands, subscription],
   );
 
+  const subscriptionName = subscription?.spec?.name;
+  const subscriptionNamespace = subscription?.metadata?.namespace;
+
+  React.useEffect(() => {
+    getOperands(subscriptionName, subscriptionNamespace)
+      .then((operandsForOperator) => {
+        if (operandsForOperator) {
+          setOperands(operandsForOperator);
+        } else {
+          setLoadOperandsErrorMessage(
```

Comment:
I'm confused why we need this case. Wouldn't any errors be handled under `catch`?

Reply:
Replaced by  `useOperandsWatcher`